proto/cosmos/group/v1/query.proto

```diff
@@ -83,6 +83,13 @@ service Query {
   rpc TallyResult(QueryTallyResultRequest) returns (QueryTallyResultResponse) {
     option (google.api.http).get = "/cosmos/group/v1/proposals/{proposal_id}/tally";
   };
+
+  // Groups queries all groups in state.
+  // 
+  // Since: cosmos-sdk 0.48
```

Comment:
@amaurym, this seems something useful to add in a point version, as it is not breaking.
How does it work with the since comment then?

Reply:
We can do `Since cosmos-sdk 0.47` (if we backport now) or `Since cosmos-sdk 0.47.1`

Reply:
Ah okay, I didn't we could do dot version. I think it's best to wait for 0.47.1 as no features should be backported now.

Reply:
Atheesh, as per this comment, could you update the sinces to `0.47.1` ?